shims/spark311/src/main/scala/com/nvidia/spark/rapids/shims/spark311/ParquetCachedBatchSerializer.scala

```diff
@@ -485,12 +483,11 @@ class ParquetCachedBatchSerializer extends CachedBatchSerializer with Arm {
       })
       cbRdd.mapPartitions(iter => CloseableColumnBatchIterator(iter))
     } else {
-      val broadcastedHadoopConf = getBroadcastedHadoopConf(conf, selectedAttributes)
```

Comment:
I will update the description. I wanted to minimize the broadcasted variables and since we are creating a new SQLConf on the executor side, it made sense to create the hadoopConf on the executor as well.